storage/core/src/main/java/com/hortonworks/registries/storage/util/StorageUtils.java

```diff
@@ -64,11 +73,44 @@ public static void ensureUnique(Storable storable,
     @SuppressWarnings("unchecked")
     public static Collection<Class<? extends Storable>> getStreamlineEntities() {
```

Comment:
@arunmahadevan Can you rename this to `getStorableEntities` or any other valid name? Lets not use streamline or registry specific names as it is common module.